src/main/java/edu/ie3/datamodel/models/timeseries/repetitive/LoadProfileEntry.java

```diff
@@ -6,52 +6,38 @@
 package edu.ie3.datamodel.models.timeseries.repetitive;
 
 import edu.ie3.datamodel.models.timeseries.TimeSeriesEntry;
-import edu.ie3.datamodel.models.value.PValue;
-import java.time.DayOfWeek;
+import edu.ie3.datamodel.models.value.load.LoadValues;
 import java.util.Objects;
 
-/** Unique entry to a {@link LoadProfileInput} */
-public class LoadProfileEntry extends TimeSeriesEntry<PValue> {
-  private final DayOfWeek dayOfWeek;
-  private final int quarterHourOfDay;
+/** Unique entry to a {@link LoadProfileTimeSeries} */
+public class LoadProfileEntry<L extends LoadValues> extends TimeSeriesEntry<L> {
+  private final int quarterHour;
```

Comment:
The dayOfWeek is used now somewhere else or not longer necessary?

Reply:
It is no longer necesary, because the implemenation of the `LoadValues` will take care of this.